priv/repo/migrations/20220513131026_create_checks.exs

```diff
@@ -6,6 +6,9 @@ defmodule SituationRoom.Repo.Migrations.SituationRoom.Site.Check do
       add :site_id, references(:sites), null: false
       add :status_code, :integer, null: false
       add :response_time, :float, null: false
+      add :type, :string, default: "head", null: false
```

Comment:
It would be better practice to create a new migration file for this instead of editing a previous one. That way we don't have to all reset our databases to run this.